modules/ROOT/pages/exchange-api.adoc

```diff
@@ -259,6 +260,50 @@ https://anypoint.mulesoft.com/exchange/api/v2/assets/:groupId/:assetId/:version/
   }'
 ----
 
+[[update-mutable-data]]
+=== Update Mutable Data
+
+This example shows how to update the mutable data of an existing asset and describe the asset in the same request. The mutable data of an asset includes categories, custom fields, tags, and documentation pages.
+
+This example publishes the category and custom field, which have the keys `categoryKey` and `fieldKey`. Publication fails if the category or the custom field is not configured for your organization.
```

Comment:
Consider reworking the topic to be something like:
In this example, you update the mutable data of an existing asset and describe the asset in the same request by <providing these values> to <achieve this result>:
I think the level of detail introducing the example is confusing. Since you already interrupt the example syntax with commentary, why not extend that practice, if you feel you need to explain each phase of the example?

Reply:
Rewrote to ensure that the text describes the example, and introduces the specific values for this specific example syntax.